src/main/java/stelnet/commodity/CommodityIntel.java

```diff
@@ -1,8 +1,5 @@
 package stelnet.commodity;
 
-import java.awt.Color;
```

Comment:
Could you change your IDE for this project, so it uses this sorting? There's going to be a lot of noise when you change sorting only for me to change it back.